src/routes/Devilstable.svelte

```diff
@@ -91,14 +92,14 @@
 </script>
 
 <Brush
-	height={mobile ? brushDimension : height}
```

Comment:
Why did you change the order? does that matter?

Reply:
for the sake of consistency